web/src/popover_menus.js

```diff
@@ -907,4 +920,265 @@ export function initialize() {
+                } else {
+                    settings_users.show_edit_user_info_modal(user_id, true);
+                }
+            });
+
+        $("body")
+            .off("click", ".sidebar-popover-mute-user")
+            .one("click", ".sidebar-popover-mute-user", (e) => {
+                const user_id = popovers.elem_to_user_id($(e.target).parents("ul"));
+                popovers.hide_all();
+
+                muted_users_ui.confirm_mute_user(user_id);
+            });
+
+        $("body").on("click", ".sidebar-popover-unmute-user", (e) => {
+            const user_id = popovers.elem_to_user_id($(e.target).parents("ul"));
+            popovers.hide_all();
+            muted_users_ui.unmute_user(user_id);
+        });
+
+        $("body").on("click", ".sidebar-popover-reactivate-user", (e) => {
+            const user_id = popovers.elem_to_user_id($(e.target).parents("ul"));
+            popovers.hide_all();
+
+            function handle_confirm() {
+                const url = "/json/users/" + encodeURIComponent(user_id) + "/reactivate";
+                channel.post({
+                    url,
+                    success() {
+                        dialog_widget.close_modal();
+                    },
+                    error(xhr) {
+                        ui_report.error(
+                            $t_html({defaultMessage: "Failed"}),
+                            xhr,
+                            $("#dialog_error"),
+                        );
+                        dialog_widget.hide_dialog_spinner();
+                    },
+                });
+            }
+            settings_users.confirm_reactivation(user_id, handle_confirm, true);
+        });
+
+        $(".simplebar-content-wrapper").on("scroll", () => {
+            instance.hide();
+        });
+    }
+
+    function get_boundary_padding() {
+        const bottomBoundary = $("body").outerHeight() - $("#compose-content").offset().top;
+        return {top: 10, bottom: bottomBoundary};
+    }
+
+    function placement_on_mobile(instance) {
+        if ($("body").outerWidth() < 576) {
+            const $overlay = $("<div>").addClass("tippy-overlay").css({
+                position: "fixed",
+                top: "0",
+                left: "0",
+                width: "100%",
+                height: "100%",
+                backgroundColor: "rgba(0, 0, 0, 0.7)",
+                zIndex: 9998,
+            });
+
+            $("body").append($overlay);
+            $overlay.on("click", () => {
+                $overlay.remove();
+            });
+            const $user_card_tippy = $(instance.popper);
+            const left = ($("body").innerWidth() - $user_card_tippy.innerWidth()) / 2;
+            const top = ($("body").outerHeight() - $user_card_tippy.outerHeight()) / 2;
+            $user_card_tippy.css({
+                transform: `translate(${left}px, ${top}px)`,
+            });
```

Comment:
This doesn't seem to be working. (Maybe due to the recent changes in main)

I don't think this is a good way to do this though. Can you look into if there are methods you can use to get the same behaviour without using a separate `div` here?